githooks/update_setup_py.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# define colors for use in output
+green='\033[0;32m'
+no_color='\033[0m'
+grey='\033[0;90m'
+
+# Jump to the current project's root directory (the one containing
+# .git/)
+ROOT_DIR=$(git rev-parse --show-toplevel || echo)
+NO_GIT=FALSE
+if [ -z "$ROOT_DIR" ]
+then
+  echo "Did not found git repository, using '$PWD' as ROOT_DIR"
+  NO_GIT=TRUE
+  ROOT_DIR=$PWD
```

Comment:
```suggestion
  echo "Did not found git repository, using '$PWD' as ROOT_DIR"
  exit 1
```
This should not happen?

Reply:
It can happen, that's why we have this exception. This script needs to work also without a .git directory. Usually, it happens if we rsync the directory to a remote machine for testing. There we don't need the .git, but potentially this script.